avail/src/main/kotlin/avail/anvil/text/DocumentPosition.kt

```diff
@@ -230,7 +230,7 @@ data class MarkToDotRange constructor(
 		{
 			"${markPosition.lineOneBased}:${markPosition.characterInLineOneBased} — " +
 				"${dotPosition.lineOneBased}:${dotPosition.characterInLineOneBased} " +
-				"(${count.absoluteValue} chars)"
+				"(${count.absoluteValue} chars @${markPosition.offset})"
```

Comment:
I know it's pedantic, but I'd like to include the special case logic for 1 character here, so we can print `char` or `chars` as appropriate.